src/main/java/uk/gov/ons/census/casesvc/logging/EventLogger.java

```diff
@@ -55,20 +56,23 @@ public void logReceiptEvent(
   }
 
   public void logRefusalEvent(
-      UacQidLink uacQidLink,
-      String eventDescription,
-      EventType eventType,
-      RefusalDTO payload,
-      EventDTO event,
-      OffsetDateTime eventMetaDataDateTime) {
+      Case caze, String eventDescription, EventType eventType, RefusalDTO payload, EventDTO event) {
 
-    logEvent(
-        uacQidLink,
-        eventDescription,
-        eventType,
-        convertObjectToJson(payload),
-        event,
-        eventMetaDataDateTime);
+    Event loggedEvent = new Event();
+
+    loggedEvent.setId(UUID.randomUUID());
+    loggedEvent.setCaze(caze);
+    loggedEvent.setCaseId(UUID.fromString(payload.getCollectionCase().getId()));
+    loggedEvent.setEventDate(event.getDateTime());
+    loggedEvent.setRmEventProcessed(OffsetDateTime.now());
+    loggedEvent.setEventDescription(eventDescription);
+    loggedEvent.setEventType(eventType);
+    loggedEvent.setEventChannel(event.getChannel());
+    loggedEvent.setEventSource(event.getSource());
+    loggedEvent.setEventTransactionId(UUID.randomUUID());
```

Comment:
this should be the transaction id that comes in on the message

Reply:
Think this should be resolved from [this bit](https://github.com/ONSdigital/census-rm-case-processor/blob/master/src/main/java/uk/gov/ons/census/casesvc/logging/EventLogger.java#L112-L116) after merge conflicts are resolved in this PR

Reply:
doesn't look this has been fixed by the conflict merges???